ansible/roles/docker/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 # role variable for the swarm service
 
 docker_api_port: 2385
+thin_provisioned: false 
```

Comment:
my next comment will explain this, but since we never actually *set* 
defaults, the variable has to be set somehow and it is not used outside 
the role. Therefore, I'm pretty sure `vars` is the best place for this, 
but I'm not going to push the issue.

Case in point: I don't want anyone to override this ever.

On 27 Feb 2016, at 15:57, Madhav Puri wrote:

>> @@ -2,3 +2,4 @@
>> # role variable for the swarm service
>>
>> docker_api_port: 2385
>> +thin_provisioned: false
>
> yes, I understand that vars can be defined at multiple places. But I 
> started keeping all vars in `defaults` so there is just one way of 
> introducing new vars. Otherwise it is hard to set a guideline for 
> adding vars especially when a var is going to have a default value. I 
> moved all vars to `defaults` for other reasons described in 
> https://github.com/contiv/ansible/issues/61.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/152/files#r54341714